docs/hil-bmi-commands.md

```diff
@@ -0,0 +1,41 @@
+# ESI Equivalency with HIL/BMI
+
+One of the goals of ESI is to replace [HIL/BML](https://github.com/CCI-MOC/hil). This document matches HIL/BMI commands with their OpenStack equivalents.
+
+* [Node Commands](#node)
+* [Network Commands](#network)
+
+## <a name="node"></a>Node Commands
+
+### Administrator
+
+* **hil node delete**: `openstack baremetal node delete`
+* **hil node nic delete/register**: ?? `openstack baremetal port create/delete`
+* **hil node register**: `openstack baremetal node create`
+
+### User
+
+* **hil node bootdev**: ??
+* **hil node console**: `openstack baremetal node console show`
+* **hil node list**: `openstack baremetal node list`
+    * showing free nodes may require ESI leasing service
+* **hil node metadata add/delete**: ??
+* **hil node network connect/detach**: will be replaced by an ESI OpenStack equivalent
+* **hil node obm disable/enable**: ??
+* **hil node power cycle/off/on/status**: `openstack baremetal node power off/on`
+* **hil node show**: `openstack baremetal node show`
+    * may need enhanced ESI version to show additional network details
+
+## <a name="network"></a>Network Commands
+
+### User
+
+* **hil network create**: `openstack network create`
+    * may need an enhanced ESI version that assigns a VLAN
+* **hil network delete**: `openstack network delete`
+* **hil network grant-access/revoke-access**: `openstack rbac create/delete`
+    * multiple OpenStack commands required; could be simplified
+* **hil network list**: `openstack network list`
+* **hil network list-attachments**: ?? is this just a list of attached nodes?
```

Comment:
yeah, what nics on what nodes are connected to a network. Sample output:
```
[naved@kzn-vbmi01res ~]$ hil network list-attachments openshift-internal
+----------+-----------+------+-------------+
|   Node   |  Project  | Nic  |   Channel   |
+----------+-----------+------+-------------+
| neu-5-10 | openshift | nic1 | vlan/native |
| neu-15-9 | openshift | nic1 | vlan/native |
| neu-3-11 | openshift | nic1 | vlan/native |
+----------+-----------+------+-------------+
```

Reply:
Okay, thanks!